File: app/src/main/java/kr/com/misemung/ui/DustPresenter.java
package kr.com.misemung.ui;

import kr.com.misemung.vo.AirInfo;

public class DustPresenter implements DustContract.UserActionsListener {

    private final DustContract.View mView;

    public DustPresenter(DustContract.View mView) {
        this.mView = mView;
    }

    @Override
    public void loadFineDustData(AirInfo airInfo, String name) {
        mView.showDustResult(airInfo, name);
    }
}

File: app/src/main/java/kr/com/misemung/ui/DustPresenter.kt
package kr.com.misemung.ui

import kr.com.misemung.ui.DustContract.UserActionsListener
import kr.com.misemung.vo.AirInfo

class DustPresenter(private val mView: DustContract.View) : UserActionsListener {
    override fun loadFineDustData(airInfo: AirInfo?, name: String?) {
        mView.showDustResult(airInfo, name)
    }
}